packages/clerk-js/src/ui.retheme/components/CreateOrganization/CreateOrganization.tsx

```diff
@@ -38,7 +38,7 @@ export const CreateOrganization = withOrganizationsEnabledGuard(
   { mode: 'redirect' },
 );
 
-export const CreateOrganizationModal = (props: CreateOrganizationProps): JSX.Element => {
+export const CreateOrganizationModal = (props: WithoutRouting<CreateOrganizationProps>): JSX.Element => {
```

Comment:
Good catch.
🙃 
Do you think that creating a type for `CreateOrganizationModal` explicitly and hide `WithoutRouting` in its definition would make this slightly more readable? I don't think there's a good enough reason to have types for the base components but not their modal versions but I'd like to hear your thoughts

Reply:
Yeah indeed that will be more readable!